sdk/template/template/karma.conf.js

```diff
@@ -1,5 +1,12 @@
 // https://github.com/karma-runner/karma-chrome-launcher
 process.env.CHROME_BIN = require("puppeteer").executablePath();
+require("dotenv").config();
+const {
+  jsonRecordingFilterFunction,
+  isPlaybackMode,
+  isSoftRecordMode,
+  isRecordMode
+} = require("@azure/test-utils-recorder");
```

Comment:
If the tests are not using the `recorder`, this is not required.
I'd recommend removing it and suggest adding in a separate PR with a couple of tests in the template package to demonstrate.

Reply:
I believe someone else is already working on standardizing testing, perhaps @sadasant. I removed the recording handling.